### DISCUSSION TITLE
How to use httpsCallable

### DISCUSSION THREAD

**Discussion by jeverduzco** [vuejs/vuefire#1278] [Questions and Help]
I have a question, with the Nuxt module can I use the [httpsCallable](https://firebase.google.com/docs/functions/callable#call_the_function) function?

**Comment by posva** [comment]
I haven't checked the functions (besides the SSR) much yet. I imagine this should just be the default for https://nuxt.com/docs/guide/directory-structure/server#server-routes when building with the firebase preset. It probably requires other work

**Comment by strid3r21** [comment]
was there ever a proper solution for this? 

being able to call custom functions is really useful. ill keep playing around with it, but for now im not having much luck
